planner/core/testdata/integration_serial_suite_out.json

```diff
@@ -102,7 +102,7 @@
       {
         "SQL": "explain format = 'verbose' select count(*) from t2 group by a",
```

Comment:
use `brief`?

Reply:
verbose conflicts with brief, you cannot set them at the same time.